src/sentry/models/dashboard.py

```diff
@@ -29,6 +29,19 @@ class Meta:
         unique_together = (("project", "dashboard"),)
 
 
+@region_silo_model
+class DashboardFavouriteUser(Model):
+    __relocation_scope__ = RelocationScope.Excluded
+
+    user = FlexibleForeignKey("sentry.User")
```

Comment:
This needs to be a hybridcloudforeignkey. The user model is in control silo, and we can't have foreign keys go between control + region mode models.